code/modules/mob/living/carbon/human/human_mob.dm

```diff
@@ -1677,49 +1677,129 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 		return
 	..()
 
+/// How long the mob has before they cannot be defibrillated back to life.
+/mob/living/carbon/human/defib_time_left()
+	if(stat != DEAD)
+		return 0
+	return ..() + revive_timer_postponement
+
+
+/mob/living/carbon/human/under_defib_timer()
+	return defib_time_left() > 0
```

Comment:
This override shouldn't be necessary, is it? The parent proc aready does the same exact check unless I'm missing something